host_applications/linux/apps/raspicam/RaspiTexStub.c

```diff
@@ -0,0 +1,240 @@
+/*
```

Comment:
I wasn't expecting a new source file based on the commit message (which has a typo, BTW). A bit more explanation would be nice.